app/(landing)/components/CTAbtn.tsx

```diff
@@ -45,12 +44,12 @@ const CTAbtn = ({
     }
 
     // Completed Form ?
-    const isProfile = session?.user?.completedProfile
+    const isProfile = session?.user?.completedProfile;
```

Comment:
Can you please add a line between `return` and the variables, just for the readablity.

```suggestion
    const isProfile = session?.user?.completedProfile;
    
```